### PR TITLE
Add Changelog

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
This PR adds a changelog. It's not the commits we make with each release, but a higher level description of what's going on.

No 4.0 changes yet, those will be added after we create the release (and be the last numbered release -- after 4.0 we can just update the changelog as needed)

### CODE REVIEW

**Review by philip-cline — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+- Add GTFS-RT vehicle overlay
+- Add user language settings
+- Support overriding strings
+
+## [3.3.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.3.0) (2021-07-22)
+
+- Field trip module
+- Add save button
+- Add delete button
```
Comment (demory) [inline CHANGELOG.md:27]: Are these specific to the field trip module or something else?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- New stop markers on map at high zoom levels
+- Add accessibility labels to legs and itineraries when accessible routing is enabled
+- Support showing multiple fare prices
+
+## [3.8.0](https://github.com/opentripplanner/otp-react-redux/releases/tag/v3.8.0) (2021-12-16)
+
+- Fix crashes reported to us
+- Improve screen reader experience
+- Add user setting for planning accessible trips by default
+- Support internationalization of most component
```
Comment (demory) [inline CHANGELOG.md:63]: 'component' -> 'components'

**Review by demory — CHANGES_REQUESTED** [review]
See inline comments

File: CHANGELOG.md (hunk 1 above)
Comment (philip-cline) [reply CHANGELOG.md]: These are specific to the field trip module, these were meant to be indented, which I fixed in https://github.com/opentripplanner/otp-react-redux/pull/537/commits/5e363e393de0502c385885d4aefd95d2d61cc484

File: CHANGELOG.md (hunk 2 above)
Comment (philip-cline) [reply CHANGELOG.md]: Good catch! Changed in https://github.com/opentripplanner/otp-react-redux/pull/537/commits/5e363e393de0502c385885d4aefd95d2d61cc484

**Review by demory — APPROVED** [review]

**Pull request merged**